api/lib/redisHelper.mjs

```diff
@@ -115,3 +115,30 @@ export async function getTotalPossibleScore() {
 export async function getMaxScores() {
     return await getStudentScores('MAX POINTS')
 }
+
+/**
+ * Gets a list of all of the students in the class.
+ * Each student is represented as a list: [legalName, email]
+ * @returns {Promise<Array<Array<string>>>} List of [legalName, email]
+ */
+export async function getStudents() {
+    const client = getClient();
+    await client.connect();
+
+    const keys = await client.keys('*');  // Retrieve all keys, adjust pattern as needed for specificity
```

Comment:
```suggestion
    const keys = await client.keys('*');
    return keys.filter((key) => key.endsWith('berkeley.edu'));
```

Reply:
can't just return the filtered keys based on the name structure, but modified method:
 let keys = await client.keys('*');  // Retrieve all keys, adjust pattern as needed for specificity
    keys = keys.filter((key) => key.includes('@'));
    const students = [];

    for (const key of keys) {
        const studentData = await client.get(key);
        if (studentData) {
            const data = JSON.parse(studentData);
            //add key.endsWith(...) to get rid of the "MAX POINTS" row
            if (data && data['Legal Name'] ) { // Ensure the data includes a legal name
                students.push([data['Legal Name'], key]); 
            }
        }
    }